README.md

```diff
@@ -339,3 +339,30 @@ The table below describes the current Landing Zone Definitions available in the
     </tr>
   </tbody>
 </table>
+
+## Testing
+
+The Landing Zone Service contains unit and integration tests. These tests are run as part of the CI pipeline during the PR process,
+as well as on merge to `main`.
+
+### Local Testing
+
+```sh
+# Unit tests
+./gradlew :service:unitTest
+
+# integration tests
+./gradlew :service:integrationTest
+```
+
+#### Setup for Local Integration Testing
```

Comment:
I tested this locally and things worked, if anyone has cycles I'd appreciate a second guinea pig for these instructions.